### DISCUSSION TITLE
Help centering button

### DISCUSSION THREAD

**Discussion by martinacantaro** [executablebooks/meta#529] [Q&A] [answered]
Hi!
I'd appreciate some help to center this button, I'm not sure what I'm doing wrong:

```
::::{div} sd-d-flex-column sd-align-major-center
:::{button-ref} learning
:color: primary

Get started!
:::
::::
```

I followed (to the best of my knowledge) the instructions in the docs site:

`Items within a flex box can also be aligned along the major axis or minor axis, dependant on the flex-direction. For example, using sd-d-flex-column and sd-align-major-center will center the items in the vertical direction.
`

I didn't edit the .css file whatsoever
Am I missing a previous step?

**Comment by martinacantaro** [comment]
Hold on, I might have setup sphinx autobuild incorrectly so it could be the case that I'm not correctly building the changes I'm making and therefore not seeing the results. I'll report back when I have more time to correct the issue

**Comment by martinacantaro** [comment, marked as answer]
Solved it! Needed to use `sd-d-flex-row` instead of `sd-d-flex-column` (and correct my autobuild configuration but that is unrelated)

**Reply by choldgraf** [reply]
ahh nice - thanks for sharing your fix!
